### ISSUE TITLE
To save view (single,facing,book) of books when I change the fit of page

### ISSUE THREAD

**Issue by Aliaksandr3by** [sumatrapdfreader/sumatrapdf#2105]
I always use the book view (ctrl+8), and sometime to change a fit two pages or continuous two pages.

But , if I click on the "fit width...", I get single page
![image](https://user-images.githubusercontent.com/11432724/128578593-3298a963-ba6b-4327-99bf-b2e6f3c6be32.png)

![image](https://user-images.githubusercontent.com/11432724/128578553-cae651eb-e877-421a-8a9e-7a2160a256f5.png)
or
![image](https://user-images.githubusercontent.com/11432724/128578560-1339cb50-5606-41bc-b983-2f0936ec821f.png)

**Comment by yashpalgoyal1304** [comment]
yeah, i was thinking about this some days ago too. So, currently, the buttons perform a combination of "toggle scroll" and "page zoom presets" - which are accessible separately under File > View. One option can be that separate UI buttons for those functionality are provided.

![image](https://user-images.githubusercontent.com/19423063/128625642-b3f7dedb-002d-449b-bc6e-3e848a193ed5.png)

**Comment by Aliaksandr3by** [comment]
I ordinarily use key 'C' for change the continuously mode, and to use ctrl+6/7/8 to change view mode.
But button on main form is useless, on my mind.
I think this is bad, and need to change.

**Comment by Aliaksandr3by** [comment]
good to add key-short. 
![image](https://user-images.githubusercontent.com/11432724/128638271-75e7da51-a03a-4c66-b89c-d6704e8d6ba6.png)

**Comment by GitHubRulesOK** [comment]
@kjk 
over the years most keys have been shown and as highlighted above this is one of any remaining exceptions

@Aliaksandr3by 
as you know icons have always been kept to the minimum,
If you want Book view 90% of the time set that in the default start-up options.
 then when you click either zoom single page keys it should toggle in and out without forgetting previous view mode.

I thought you use AHK so could remap some key sequences to suit your preference.

**Comment by GitHubRulesOK** [comment]
@kjk  @Aliaksandr3by 
recent menu changes mean you can add controls as text icons or reduce the keys required by assign single commands to one key
you still need 2 keys for 2 commands (though you can often run a script to call/chain 2 functions via -DDE command line)
as the main issue is the menus and have changed I suggest this issue is no longer unique.

<img width="463" height="376" alt="Image" src="https://github.com/user-attachments/assets/842212aa-0817-4bc4-bfec-d162be441ab8" />
